### ISSUE TITLE
Raw Data Access

### ISSUE THREAD

**Issue by cpsubrian** [propublica/congress-api-docs#52]
Some of the issues I've been running into over the past couple days could be avoided if I had access to the raw data (like how I could rsync from govtrack). I totally understand that your goal is to create a useful REST API for previous consumers of the govtrack and sunlight APIs, but I wonder if there is room here for access to the pre-scraped, pre-parsed data also.

Perhaps an S3 bucket or some other cloud resource that could be synced with?

**Comment by dwillis** [comment]
@cpsubrian do you mean something like what's available here? https://github.com/unitedstates/congress/wiki

**Comment by cpsubrian** [comment]
@dwillis Indeed but without having to run the scrapers myself. The last time I tried it took > 6 hours and I ended up not even being able to parse the data (I don't remember the exact errors). In a nutshell, if ProPublica is running those (or similar) scrapers to keep up-to-date, and perhaps even merging in data from other sources, it would be amazing to expose the net result in a sharable way (read-only would be fine).

Though your API is shaping up to be a great resource, some people may prefer to load subsets into ElasticSearch, or Mongo, or DynamoDB, etc. etc..

Since (maybe I'm assuming wrong here) there is no profit-motive for ProPublica surrounding this data, I would think exposing more ways to consume would be a win for open data. Granted, I understand there is a cost to hosting large amounts of files, though perhaps something along the lines of Google's open BigTable datasets is the right direction.
